src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/bookshop";
+    private static final Properties DB_PROPERTIES;
+    private static final String DB_USER = "root";
+    private static final String DB_PASSWORD = "Wetuop34!";
+    private static final String DB_DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String USER_KEY = "user";
+    private static final String PASSWORD_KEY = "password";
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put(USER_KEY, DB_USER);
+        DB_PROPERTIES.put(PASSWORD_KEY, DB_PASSWORD);
+
+        try {
+            Class.forName(DB_DRIVER);
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't load MSQL driver" + e);
```

Comment:
There is a typo in the exception message: 'Can't load MSQL driver'. It should be 'Can't load MySQL driver'. Consider correcting this for clarity.